### ISSUE TITLE
Possible to dockerise this?

### ISSUE THREAD

**Issue by psychemedia** [RinkeHoekstra/gephi-servlet#1]
I never really got to grips with the Gephi toolkit, but as I start to explore the world of docker, I've been wondering: would it be possible to run this servlet as a service in a docker container, perhaps passing the graph file URL via a parameter and getting something like SVG or in return?

If so, I guess it would also be useful to be able to pass layout settings into via a URL parameters?

**Comment by RinkeHoekstra** [comment]
Makes sense, but this is particular servlet is really old stuff and limited in functionality. I guess it would make more sense to use e.g. the d3.js library of Mike Bostock (http://d3js.org).

I have no plans to revive this work, but feel free to fork it.

**Comment by psychemedia** [comment]
Beyond my ken at the moment... I tend to avoid Java...;-)
